src/commands/sdk/execute.ts

```diff
@@ -69,33 +71,70 @@ Other
 
   static readStdin = true;
 
+  async toJavascript(filename: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      exec(
+        `npx tsc -t es5 ${filename}`,
+        () => {
+          // We do not care for TS errors, user should now if this will work or not
+          resolve(true);
+        });
+    });
+  }
+
   async beforeConnect() {
     this.code = this.stdin || this.args.code || "// paste your code here";
+    try {
+      eval(this.code);
```

Comment:
If possible, it would be preferable to attempt evaluating in a separate V8 context
https://nodejs.org/api/vm.html#vmruninnewcontextcode-contextobject-options